### ISSUE TITLE
UI problem with standard mods 

### ISSUE THREAD

**Issue by ViggiSmallz** [LoneGazebo/Community-Patch-DLL#4958]
_1. Mod version (i.e Date - 4/23):_
**3.11.21**

_2. Mod list (if using Vox Populi only, leave blank):_

**https://i.imgur.com/hiqK3Uz.png**

_3. Error description:_ 
**UI ERROR**

_4. Steps to reproduce (optional):_

**all files are unedited fresh download via steam or civfanatics**

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: 
http://forums.civfanatics.com/showthread.php?t=487482):

**http://www.mediafire.com/file/d4ffjqd7mk3rlke/mods.zip/file (not sure if i did this right but i hope it helps)**

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

**http://www.mediafire.com/file/cu0dcd8tz9a28d0/Augustus_Caesar_0192_AD-1320.Civ5Save/file**

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):
4. Screenshots (optional):

**https://imgur.com/a/GjfqFVV**

**Comment by ViggiSmallz** [comment]
is there anyone that can help? what version should i use theyre supposed to work together

**Comment by bigcat88** [comment]
Unsubscribe from all steam mods(including VOX). Folder 'MODS' must be clear.
1. Install only this: https://mega.nz/#F!DMsxkDYI!zSpR9CHpmAfWyIvMXaN3pw. 
Not working -> invalid version of game or missing DLC.
2. Try to add mods one by one. Check game every time. Not working -> post mod name.
3. Repeat step 2.

**Comment by Vyyt02** [comment]
Yeah, your other mods seem harmless, but as the bigcat88 says, the first step should be trying the game only with VP. If you do not get the glitches, then you know you are having a mod conflict.
